CityPuzzle/Game Room/Create GameRoom/CreateGamePage.xaml

```diff
@@ -24,8 +25,10 @@
                 </Grid.ColumnDefinitions>
                 <Button x:Name="addobj" Text="Sudaryti uzduotis" Clicked="AddObj_click" BackgroundColor="#F3BD21" IsVisible="False" Grid.Row="1" Grid.Column="1" CornerRadius="15"></Button>
             <Button x:Name="lookobj" Text="Perziureti uzduotis" Clicked="Look_click" BackgroundColor="#F3BD21" IsVisible="False" Grid.Row="3" Grid.Column="1" CornerRadius="15"></Button>
-            <Button x:Name="approved" Text="Issaugoti" Clicked="Addgamer_click" BackgroundColor="#F3BD21"  IsVisible="False" Grid.Row="5" Grid.Column="1" CornerRadius="15"></Button>
-            </Grid>
+                <Button x:Name="approved" Text="Issaugoti" Clicked="Addgamer_click" BackgroundColor="#F3BD21"  IsVisible="False" Grid.Row="5" Grid.Column="1" CornerRadius="15"></Button>
```

Comment:
In multiword names every word should start with an uppercase letter except the first one. First letter deppends on the situation. Please fix button names 'addObj', 'lookObj' and 'AddGamer_click'. Also make more descriptive name for clicked event like 'SaveObj_clicked'